ci/common-ci.yml

```
@@ -74,6 +74,11 @@ variables:
   variables:
     COMMON_SPACK_ENVIRONMENT: ci/docker/common.yaml
 
+.build_deps_common_mi:
```

Purely subjective, but would something like this be clearer:
```suggestion
.build_deps_common_amdgpu:
```
?